cmd/atlas/internal/cloudapi/graphql/client.go

```diff
@@ -0,0 +1,244 @@
+// The code in this package is taken verbatim from https://github.com/Khan/genqlient/blob/main/graphql/client.go
+// to avoid a dependency on genqlient.
+
+package graphql
```

Comment:
maybe keep this inside cloudapi; the path feels too long. It can also be a private thing